GeosCore/sulfate_mod.F90

```diff
@@ -1045,17 +1045,18 @@ SUBROUTINE SRCSF30( Input_Opt, State_Grid, State_Met, State_Chm, TC2, RC )
     AREA = HcoState%Grid%AREA_M2%Val(:,:)
     !AREA3D(:,:,1) = AREA(:,:)
     !AREA3D(:,:,2) = AREA(:,:)
-
+    
+     ! comment out if shut off subgridcoag bc,14/12/23
```

Comment:
Rather than commenting this out, I would recommend bracketing this code with `IF ( Input_Opt%SGCOAG ) THEN... ENDIF`.

Reply:
Response to this comment is given above. The 'true' option could give erroneous results - I advise against introducing this input option until a development is made to address potentially erroneous resolution-dependent functionality of SGCOAG in its current state.